homeassistant/components/climate/__init__.py

```diff
@@ -379,7 +379,7 @@ def _report_turn_on_off(feature: str, method: str) -> None:
 
         supported_features = self.supported_features
         if supported_features & (
-            ClimateEntityFeature.TURN_ON | ClimateEntityFeature.TURN_ON
+            ClimateEntityFeature.TURN_ON | ClimateEntityFeature.TURN_OFF
```

Comment:
Considering making this a constant so the union doesnt have to run every time and construct a new object

Reply:
https://github.com/home-assistant/core/pull/121619